### PR TITLE
Add LiDAR visualization w. example

### PR DESCRIPTION
**Pull request by casparvv**
Adds LiDAR detection visualization and an example for the point robot.
Also fixes a calculation error in the LiDAR processing.

### CODE REVIEW

File: examples/point_robot_lidar_sensor.py [hunk 1]
```diff
@@ -30,6 +33,11 @@ def run_point_robot_with_lidar(n_steps=1000, render=False, obstacles=True, goal=
     history = []
     for _ in range(n_steps):
         ob, _, _, _ = env.step(action)
+        
+        # Visualize LiDAR ray detections as spheres
```
Comment (maxspahn) [inline examples/point_robot_lidar_sensor.py:37]: This comment must still be removed.

File: urdfenvs/sensors/lidar.py [hunk 2]
```diff
@@ -2,8 +2,10 @@
 import numpy as np
 import pybullet as p
 import gym
+from typing import List
 
 from urdfenvs.sensors.sensor import Sensor
+from urdfenvs.urdf_common.generic_robot import GenericRobot
```
Comment (maxspahn) [inline urdfenvs/sensors/lidar.py:8]: This import is not needed.

**Comment by maxspahn** [comment]
@casparvv Thank you very much for this needed visualization of the lidar sensor.

I changed the location of your visualization function and slightly improved the code style.

File: urdfenvs/sensors/lidar.py [hunk 3]
```diff
@@ -68,14 +71,62 @@ def sense(self, robot, *args):
...
+            self._rel_positions[2 * i : 2 * i + 2] = lidar[0][2] * self._ray_length * np.array([np.cos(theta + alpha), np.sin(theta + gamma)])
             self._distances[i] = np.linalg.norm(self._rel_positions[2 * i: 2 * i +2])
+        self.update_lidar_spheres(lidar_position)
         if self._raw_data:
-            return {'rays': self._distances}
-        return {'rays': self._rel_positions}
+            return self._distances
+        return self._rel_positions
+
+    def init_lidar_spheres(self, lidar_position):
```
[the remark this answers is not in the document]
Comment (casparvv) [reply urdfenvs/sensors/lidar.py:92]: Nice, that makes sense

**Pull request merged**